packages/perseus/src/widgets/grapher/util.tsx

```diff
@@ -669,13 +722,15 @@ export const defaultPlotProps = (type: string, graph: any): any => {
     // widget before even reading the question; you can't lose, but you
     // might get a free win.
     const model = functionForType(type);
+    const {defaultAsymptote} =
+        "defaultAsymptote" in model ? model : {defaultAsymptote: null};
```

Comment:
Does this work?

```suggestion
    const defaultAsymptote = "defaultAsymptote" in model ? model.defaultAsymptote : null;
```

Reply:
This line is object destructuring, so I think the brackets are needed (model refers to the asymptotic graph objects). Tried your suggested change and TS errors popped up.

Reply:
That's interesting, when I pull your branch and make the change I don't get errors. I don't see the point of making an object just to pull `null` out of it.

Reply:
Hmm, oh, I think I didn't fully read the change the first time. Updating it :)